external-crates/move/move-stdlib/sources/error.move

```diff
@@ -48,9 +48,6 @@ module std::error {
   /// Out of gas or other forms of quota (http: 429)
   const RESOURCE_EXHAUSTED: u64 = 0x9;
 
-  /// Request cancelled by the client (http: 499)
-  const CANCELLED: u64 = 0xA;
```

Comment:
Possible we want to mark this as unused for now, since it is setting out expected error cods for future use.